lib/integrations/awsoidc/testdata/TestAccessGraphAWSIAMConfigOuput.golden

```diff
@@ -0,0 +1,65 @@
+"access-graph-aws-iam" will perform the following actions:
+
+Attach an inline IAM policy named "AccessGraphSyncAccess" to IAM role "integrationrole".
+PutRolePolicy: {
+    "PolicyDocument": {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": [
```

Comment:
actually idk what the purposes of these actions are and it's out of scope for the PR to update the statements,
but from a cursory look, I could just say "Sid: sync ec2 resources", "Sid: sync RDS resources", I guess?

Let's just make an issue for TAG team to update this since they made the policy

Reply:
or instead of an issue, cc @tigrato what do you think about grouping these statement actions and adding a descriptive SID to each group?